src/pages/ResearcherConsole.js

```diff
@@ -152,17 +112,27 @@ export default function ResearcherConsole() {
     }
   };
 
+
+  //Draft delete, by referenceIds
+  const deleteDraftById = async ({ referenceId, collectionsClone }) => {
+    await DAR.deleteDar(referenceId);
+    const targetIndex = findIndex((draft) => {
+      return draft.referenceId === referenceId;
+    })(collectionsClone);
+
+    return targetIndex;
+  };
+
   //Draft delete, passed down to draft table to be used with delete button
-  const deleteDraft = async ({ referenceId, identifier }) => {
+  const deleteDraft = async ({ referenceIds, identifier }) => {
     try {
-      await DAR.deleteDar(referenceId);
       const collectionsClone = cloneDeep(researcherCollections);
-      const targetIndex = findIndex((draft) => {
-        return draft.referenceId === referenceId;
-      })(collectionsClone);
+      const targetIndex = deleteDraftById({ referenceId: referenceIds[0] });
```

Comment:
provide `collectionsClone` when calling `deleteDraftById` (or you could inline `deleteDraftById` here since we're no longer iterating through `referenceIds`)

Reply:
I will provide `collectionsClone` because I kind of like pulling it out into a separate function :D